polaris-service/src/main/java/org/apache/polaris/service/PolarisApplication.java

```diff
@@ -288,7 +289,7 @@ public void run(PolarisApplicationConfig configuration, Environment environment)
     // different processes
     // and in-memory state will be lost b/w invocation of bootstrap command and running a server
```

Comment:
Nit: Can we format the comment a bit since we are here. Not a blocker though.

Reply:
I reverted the change so it's not updated anymore. I will update for now.